moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h

```diff
@@ -165,7 +165,7 @@ class PlannerManager
   /// It is assumed that motion plans will be computed for the robot described by \e model and that any exposed ROS
   /// functionality
   /// or required ROS parameters are namespaced by \e ns
```

Comment:
Update the docstrings where appropriate

Reply:
I think is the only header that has changed

Reply:
friendly ping @mlautman